shared/src/business/useCaseHelper/consolidatedCases/formatConsolidatedCaseCoversheetData.js

```diff
@@ -0,0 +1,48 @@
+const { Case } = require('../../entities/cases/Case');
+
+/**
+ * Formats consolidated cases coversheet data
+ *
+ * @param {object} providers.applicationContext the applicationContext
+ * @param {object} providers.caseEntity the case entity
+ * @param {object} providers.coverSheetData the coversheet data
+ * @param {object} providers.docketEntryEntity the docketEntryEntity
+ * @returns {object} coversheet data with consolidated cases
+ */
+exports.formatConsolidatedCaseCoversheetData = async ({
```

Comment:
💭  I think elsewhere we've begun removing sorting and filtering from the `format` function, and this doesn't seem to be doing any actual formatting, so I think we should consider renaming this function.

Reply:
or maybe an example of a file where it's doing what you're referencing could be helpful

Reply:
I believe `web-client/src/presenter/computeds/formattedMessages.js` used to do sorting, filtering, and formatting, and we pulled out the sorting/filtering.